<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InvoiceDetailWindow</class>
 <widget class="QDialog" name="InvoiceDetailWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#VAT, #detailId, #facilityName, #facilityQuantity, #invoiceDetailPrice {
	background-color: #00A4E8;
	color: #ffffff;
	font: Bold &quot;Inter&quot;;
}

QTableWidget
{
	background-color: #fff;
}

QPushButton {
	border-radius: 5px;
	height: 24px;
	font-weight: bold;
	color: #ffffff;
}

#invoiceDetailAddButton
{
	background-color: #198754;
}

#invoiceDetailAddButton:hover
{
	background-color:  #157347;
}

#invoiceDetailDeleteButton
{
	background-color: #dc3545;
}

#invoiceDetailDeleteButton:hover
{
	background-color: #bb2d3b;
}

#invoiceDetailEditButton
{
	background-color: #0d6efd;
}

#invoiceDetailEditButton:hover
{
	background-color: #0b5ed7;
}

QDialog
{
	background-color: #fff;
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>711</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QLabel" name="detailId">
      <property name="styleSheet">
       <string notr="true">#label {
	background-color: #00A4E8;
	color: #ffffff;
	font: Bold &quot;Inter&quot;;
}</string>
      </property>
      <property name="text">
       <string>Mã</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="facilityName">
      <property name="styleSheet">
       <string notr="true">#label {
	background-color: #00A4E8;
	color: #ffffff;
	font: Bold &quot;Inter&quot;;
}</string>
      </property>
      <property name="text">
       <string>Tên Vật tư</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="facilityQuantity">
      <property name="styleSheet">
       <string notr="true">#label {
	background-color: #00A4E8;
	color: #ffffff;
	font: Bold &quot;Inter&quot;;
}</string>
      </property>
      <property name="text">
       <string>Số lượng</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="invoiceDetailPrice">
      <property name="styleSheet">
       <string notr="true">#label {
	background-color: #00A4E8;
	color: #ffffff;
	font: Bold &quot;Inter&quot;;
}</string>
      </property>
      <property name="text">
       <string>Đơn giá</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="VAT">
      <property name="styleSheet">
       <string notr="true">#label {
	background-color: #00A4E8;
	color: #ffffff;
	font: Bold &quot;Inter&quot;;
}</string>
      </property>
      <property name="text">
       <string>VAT</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="invoiceDetailTable">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>711</width>
     <height>521</height>
    </rect>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::PenStyle::DashLine</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="InvoiceGroup">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>571</height>
    </rect>
   </property>
   <property name="title">
    <string>Thông tin chi tiết hóa đơn</string>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mã</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="invoiceDetailId">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>vật tư</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QPushButton" name="invoiceDetailAddButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Thêm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="invoiceDetailDeleteButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Xóa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="invoiceDetailEditButton">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Hiệu chỉnh</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="facilityQuantityText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Số lượng</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="invoiceDetailPriceText">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Đơn giá</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>VAT</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="vatPrice">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="acceptRichText">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="facilityNameBox">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>171</width>
      <height>24</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
